src/icons/autocad/IconAutoCAD.js

```diff
@@ -0,0 +1,176 @@
+// @flow
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+type Props = {
+    className?: string,
+    height?: number,
+    /** A text-only string describing the icon if it's not purely decorative for accessibility */
+    title?: string | React.Element<any>,
+    width?: number,
+};
+
+const IconAutoCAD = ({ className = '', height = 30, title, width = 30 }: Props) => (
```

Comment:
I feel that this svg can likely be cleaned up. Has a lot of defs and urls.

Reply:
sure, i will check with Zach on that, this is the raw data we are given right now

Reply:
You may be able to run it through an SVG optimizer without affecting the image.